packages/amplify-cli/src/utils/global-prefix.ts

```diff
@@ -21,3 +25,62 @@ function getYarnPrefix() {
   }
   return yarnPrefix;
 }
+
+// NOTE: This issue: https://github.com/jonschlinkert/global-prefix/issues/30 prevents us from using the global-prefix package
+function getGlobalNpmPrefix() {
+  // System-wide prefix
+  if (process.env.PREFIX) return process.env.PREFIX;
+
+  // $HOME variable
+  const home = os.homedir();
```

Comment:
for the various variables that represent a path, can you append "Path" to the end for readability?

```ts
  const homeDirPath = os.homedir();
```